src/__tests__/extractMeta.js

```diff
@@ -203,6 +203,50 @@ describe('Metadata extractor', () => {
         });
     });
 
+    it('Extracts metadata from date type', () => {
+        const props = {
+            dateProp: ElementPropTypes.date,
+            datePropIsRequired: ElementPropTypes.date.isRequired
```

Comment:
```suggestion
            datePropRequired: ElementPropTypes.date.isRequired
```
this would be more consistent with the ones below. Looks like this test is checking that backwards compatibility thing parsing props that don't have a proper `{ label: "", type: ElementPropTypes. }`.